react/components/utils/zIndex.ts

```diff
@@ -0,0 +1,4 @@
+export default {
+  modal: 10000,
+  tooltip: 11000,
```

Comment:
Not sure though that those values have to be this high.

Reply:
Me neither. Actually, we could follow something [like this](https://material-ui.com/customization/z-index/). I didn't change the modal `zIndex` because I don't really know if something would break.

Reply:
Interesting, it could be of some help yes.